OpenSim/Actuators/TorqueActuator.cpp

```diff
@@ -85,24 +85,21 @@ void TorqueActuator::constructProperties()
 // BodyID
 //-----------------------------------------------------------------------------
 //_____________________________________________________________________________
-/**
- * Set the generalized Body to which the Body actuator is applied.
- *
- * @param aBody Pointer to the generalized Body.
+/*
+ * Set the PhysicalFrame to which the Body actuator is applied.
+
  */
-void TorqueActuator::setBodyA(const Body& aBody)
+void TorqueActuator::setBodyA(const PhysicalFrame& aBody)
```

Comment:
> aFrame, not aBody

While you're at it, get rid of the "a" from the formal parameter name -- should be just "frame".